src/js/utils/html-utils.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Useful functions for dealing with HTML.
+ *
+ * In particular, these functions default to being safe against
+ * Cross Site Scripting (XSS) attacks.
+ */
+(function(define) {
+    'use strict';
+    define(['underscore'], function(_) {
+        var interpolate, escapeInterpolateParameters, HTML;
+
+        /**
+         * Creates an HTML snippet.
+         *
+         * The intention of an HTML snippet is to capture a string that
+         * is known to contain HTML that has been safely escaped.
+         * This allows interpolate to understand that it does not
```

Comment:
Possible adjustment of "This allows interpolate to..." to "As an example, this allows interpolate to...".  This might clarify that other outside code could have the same understanding as interpolate.